### PR TITLE
feat: add cspell

### PR DESCRIPTION
**Pull request by kupuma-ru21**
https://github.com/graphql/graphiql/issues/2135

added cspell

### CODE REVIEW

**Comment by acao** [comment]
thank you @kupuma-ru21 !

**Comment by kupuma-ru21** [comment]
> thank you @kupuma-ru21 !

@acao 

I'm not very good at English.
So there were many words that I couldn't decide if they were typos or not.

I fixed the words that were obviously typos, but I defined the other words in "custom-words.txt".

I would appreciate it if you could give me feedback.

**Review by acao — COMMENTED** [review]
Oops sorry! thought I'd submitted this review a while ago.

File: custom-words.txt [hunk 1]
```diff
@@ -0,0 +1,280 @@
+PAAS
```
[the remark this answers is not in the document]
Comment (kupuma-ru21) [reply custom-words.txt:1]: Sorry, I just saw that.
I can commit next weekend!

**Comment by acao** [comment]
awesome, thank you!

**Comment by acao** [comment]
@kupuma-ru21 i have your branch updated and ready to go if you can allow me to push to it!

**Comment by thomasheyenbrock** [comment]
I rebased the branch, fixed some more typos and pushed everything to #2512. Closing this PR in favour of the new one. Thanks a lot @kupuma-ru21 for this! 🙏 

@acao if you have more changes that are still valid you should be able to easily rebase them on top of `main` (I reused the same commits as this PR)

**Pull request closed without merging**